Cargo.toml

```diff
@@ -2,14 +2,11 @@
 members = ["milli", "http-ui", "benchmarks", "infos", "helpers", "search"]
 default-members = ["milli"]
 
-[profile.dev]
-opt-level = 3
```

Comment:
@irevoire, should we keep this?

Reply:
If it's not taking too much time, yes we should :slightly_smiling_face: